content/application.md

```diff
@@ -40,8 +40,11 @@ As soon as a match occurs, the network manager will send mails to the mentee(s)
 
 ## How to apply
 
-1. Click on the application link to fill the form.
-2. Answer all questions (Tip: be yourself, we want to know you and what you want to achieve with this mentorship).
-3. Review your answers and click submit.
+1.  Check the application template:
+    - (Mentee Application Template)[/content/mentee_application/]
```

Comment:
Without `/content`

Reply:
Yes, it shows localhost:8080/mentee_application. Is that right?